source/common/json/json_streamer.h

```diff
@@ -409,5 +424,10 @@ class Streamer {
 #endif
 };
 
+/**
+ * A Streamer that streams to a Buffer::Instance.
```

Comment:
I think it's OK, if you want, to drop this nickname in a future PR, and change all the refs in admin stats. But for this PR this is good. I don't want you to add files to this PR.